src/test/java/org/broadinstitute/hellbender/tools/GatherVcfsCloudIntegrationTest.java

```diff
@@ -143,7 +149,7 @@ public void testBlockGather(final File vcf, final int numShards) throws IOExcept
                 VariantContextTestUtils.assertEqualVariants(actual, expected);
 
                 Assert.assertEquals(((VCFHeader) outputDataSource.getHeader()).getMetaDataInInputOrder(),
-                                    ((VCFHeader) input.getHeader()).getMetaDataInInputOrder());
+                        ((VCFHeader) input.getHeader()).getMetaDataInInputOrder());
```

Comment:
Your IDE seems to have reformatted a lot of the whitespace in this file...you should try to avoid introducing whitespace changes into your diffs unless they're to fix actual formatting issues.

Reply:
Yea, that was accidental, though i'm reasonably sure all of those cases involved reformatting whitespace from tabs to spaces which is a good thing.